autopush/tests/test_main.py

```diff
@@ -0,0 +1,84 @@
+import os
+import unittest
+
+import twisted.trial
+import twisted.internet
+from mock import Mock, patch
+from moto import mock_dynamodb2
+from nose.tools import eq_
+from twisted.python import log
+
+from autopush.main import connection_main, endpoint_main, unified_setup
+
+mock_dynamodb2 = mock_dynamodb2()
+
+
+def setUp():
+    mock_dynamodb2.start()
+
+
+def tearDown():
+    mock_dynamodb2.stop()
+
+
+class SentryLogTestCase(twisted.trial.unittest.TestCase):
+    def setUp(self):
+        twisted.internet.base.DelayedCall.debug = True
+        raven_patcher = patch("autopush.main.raven")
+        self.mock_raven = raven_patcher.start()
```

Comment:
Ah, I tried using the `@patch` decorator in #5 to override Twisted's error logger, but it didn't do anything. Had to use Trial's `patch` method instead. Didn't try creating a patcher like this, though.